src/storage/src/manifest/storage.rs

```diff
@@ -94,32 +125,56 @@ impl LogIterator for ObjectStoreLogIterator {
 #[derive(Clone, Debug)]
 pub struct ManifestObjectStore {
     object_store: ObjectStore,
+    compress_type: CompressionType,
     path: String,
 }
 
 impl ManifestObjectStore {
     pub fn new(path: &str, object_store: ObjectStore) -> Self {
         Self {
             object_store,
+            compress_type: DEFAULT_COMPRESSION_TYPE,
```

Comment:
Please add a TODO here: make it configurable.

Reply:
done